app/src/handlers/census/AchievementEarnedHandler.ts

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
Can you move the comments detailing the census events structure out of the handlers and into the event classes themselves, as we don't fully need to know the structure of the census event at the handler level as it's actually parses at the event level.

Reply:
Done